packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import { ChartConfig } from '../../hooks/useChartConfig';
+import {
+    BigNumber,
+    BigNumberContainer,
+    BigNumberLabel,
+    SimpleStatisticsWrapper,
+} from './SimpleStatistics.styles';
+
+interface Props {
+    data: ChartConfig;
+}
+
+const SimpleStatistic: FC<Props> = ({ data }) => {
+    if (!data.plotData) return null;
+    const metricKey = data.series[0];
+    const bigNumberLabel = data.metricOptions[0].label;
+
+    const bigNumber = Math.max(...data.plotData.map((o) => o[metricKey]));
```

Comment:
@nathaliabj does this take the biggest value in the column?

@TuringLovesDeathMetal should this be just the first number on the first row in that column?

Reply:
Thought it was the biggest number in the metric 🙈  Just updated it!